docs/src/main/sphinx/release/release-374.md

```diff
@@ -0,0 +1,81 @@
+# Release 374 (16 Mar 2022)
+
+## General
+
+* Implement parallel read for S3-based spooling in
```

Comment:
Improve performance when reading from S3-based spool for fault tolerant execution.